core/src/main/scala/io/dantb/contentless/RichText.scala

```diff
@@ -1,5 +1,47 @@
 package io.dantb.contentless
 
+enum RichTextNodeType(val asString: String):
+  case Heading1        extends RichTextNodeType("heading-1")
+  case Heading2        extends RichTextNodeType("heading-2")
+  case Heading3        extends RichTextNodeType("heading-3")
+  case Heading4        extends RichTextNodeType("heading-4")
+  case Heading5        extends RichTextNodeType("heading-5")
+  case Heading6        extends RichTextNodeType("heading-6")
+  case Quote           extends RichTextNodeType("blockquote")
+  case Hr              extends RichTextNodeType("hr")
+  case OrderedList     extends RichTextNodeType("ordered-list")
+  case UnorderedList   extends RichTextNodeType("unordered-list")
+  case Hyperlink       extends RichTextNodeType("hyperlink")
+  case EntryLinkInline extends RichTextNodeType("embedded-entry-inline")
+  case EntryHyperlink  extends RichTextNodeType("entry-hyperlink")
+  case EntryLinkBlock  extends RichTextNodeType("embedded-entry-block")
+  case AssetHyperlink  extends RichTextNodeType("asset-hyperlink")
+  case AssetLinkBlock  extends RichTextNodeType("embedded-asset-block")
+  case Table           extends RichTextNodeType("table")
+
+object RichTextNodeType:
+  import RichTextNodeType.*
+  val All: Set[RichTextNodeType] = Set(
```

Comment:
I think in Scala 3 `enum`s already define an `All` for you, forgot what it's called

Reply:
Thanks! Still forget, it can be used when they're all singleton objects